manifests/6_grafana.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: grafana-core
+  namespace: prombench-{{ .PR_NUMBER }}
+  labels:
+    app: grafana
+    component: core
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: grafana
+        component: core
+    spec:
+      serviceAccountName: prometheus
+      containers:
+      - image: grafana/grafana:5.0.0
+        name: grafana-core
+        imagePullPolicy: IfNotPresent
+        # resources:
+        #   # keep request = limit to keep this container in guaranteed class
+        #   limits:
+        #     cpu: 100m
+        #     memory: 100Mi
+        #   requests:
+        #     cpu: 100m
+        #     memory: 100Mi
+        env:
+          # The following env variables set up basic auth twith the default admin user and admin password.
+          - name: GF_PATHS_PROVISIONING
+            value: "/opt/grafana-provision-files"
+          - name: GF_SECURITY_ADMIN_PASSWORD
+            value: admin
+          - name: GF_SERVER_ROOT_URL
+            value: "http://{{.INGRESS_IP}}/prombench-{{ .PR_NUMBER }}/grafana/"
```

Comment:
where do you set the INGRESS_IP var?

Reply:
There are 2 ways of setting this
- we save ingress-ip in the prow config & prow saves this as an environment variable in the prowjob
- (preferred) we use a hostname in-place of INGRESS_IP, which will be resolved by an external DNS.

Reply:
will this need to be manually set or we query this from some API?

Reply:
I prefer using a hostname, something like `prombench.prometheus.io`
With this just we just need to set the INGRESS_IP at the DNS server.
So if for some reason, cluster needs to be redeployed & INGRESS_IP changes, there will be no need to change the prow-config